manifests/init.pp

```diff
@@ -156,12 +159,14 @@
   }
 
   class { 'katello::globals':
-    enable_ostree => $enable_ostree,
-    enable_yum    => $enable_yum,
-    enable_file   => $enable_file,
-    enable_puppet => $enable_puppet,
-    enable_docker => $enable_docker,
-    enable_deb    => $enable_deb,
+    enable_ostree         => $enable_ostree,
+    enable_yum            => $enable_yum,
+    enable_file           => $enable_file,
+    enable_puppet         => $enable_puppet,
+    enable_docker         => $enable_docker,
+    enable_deb            => $enable_deb,
+    use_pulp_2_for_file   => $use_pulp_2_for_file,
```

Comment:
These probably need to be parameters to `katello::application`, not globals.

Reply:
I honestly have no idea what's what anymore. So if thats the best place for this I'll do whatever. `katello::application` includes params which inherits globals. So when and how do you know where to pass what?

Reply:
I should write this down in the README, but the general rule is that `katello::globals` affects multiple services that a user is expected to be able to change (https://github.com/theforeman/foreman-installer/pull/421 would expose it to the end user). `katello::params` is intended for advanced uses and the installer wouldn't expose them. Then we have the individual classes (`katello::application`, `katello::candlepin`, `katello::pulp` and `katello::qpid`) which are directly exposed to the user as top level parameters. The class `katello` is for compatibility and going away. Right now it's there not to break the installer experience. The reason it's still here is more for downstream than upstream.